strings/base_implements.h

```diff
@@ -430,6 +430,15 @@ namespace winrt::impl
         }
     };
 
+    template <>
+    struct runtime_class_name<Windows::Foundation::IInspectable>
```

Comment:
Note that `name_of` still returns `Object`, which is necessary for proper autogeneration of parameterized interfaces.

Reply:
Yep, that's kinda important. 😉